automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/ClusterMonitoringSpec.scala

```diff
@@ -60,6 +64,66 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
       }
     }
 
+    // PetNotebookServiceAccountProvider edition.  IGNORE.
+    "should create a cluster in a different billing project using PetNotebookServiceAccountProvider and put the pet's credentials on the cluster" ignore withWebDriver { implicit driver =>
```

Comment:
Manually confirmed: all automation tests pass when un-ignoring this (and ignoring the above) and running against a FIAB with https://github.com/broadinstitute/firecloud-develop/pull/963

Reply:
imo it'd be cool to check in that conf change, but commented out, and a note on "here's another implementation you can switch between"

Reply:
I like that idea.